text/0010-trits-module/0010-trits-module.md

```diff
@@ -0,0 +1,125 @@
++ Feature name: `trits-module`
++ Start date: 2019-09-20
++ RFC PR: [iotaledger/bee-rfcs#10](https://github.com/iotaledger/bee-rfcs/pull/10)
++ Bee issue: [iotaledger/bee#45](https://github.com/iotaledger/bee/issues/45)
+
+# Summary
+
+This RFC proposes a `trits` interface to standardise the usage of different underlying encodings and their conversions from one to another.
+
+# Motivation
+
+A `trit` is a digit in base `3` and is often represented by the set `{-1, 0, +1}`. This representation is called balanced ternary and the object represented in such a way a `balanced trit`. Further reading: http://homepage.divms.uiowa.edu/~jones/ternary/bct.shtml.
+
+To work with trits on a binary machine, the most apparent solution is to encode them with the mapping `one balanced trit -> one signed byte`. This, however, is not very memory efficient, because a `byte` can represent `256` different values, while a `trit` only represents `3`.
```

Comment:
```suggestion
To work with trits on a binary machine, the most apparent solution is to encode them with the mapping `one balanced trit -> one signed byte`. This, however, is not very memory efficient, because a `byte` can represent `256` different values and is able to store multiple trits through encoding methods.
```